tests/unit/models/test_projects.py

```diff
@@ -1969,6 +1974,74 @@ def test_build_planner_get_build_plan(platforms, expected_build_infos):
     assert actual_build_infos == expected_build_infos
 
 
+@pytest.mark.parametrize(
+    ("architectures", "expected_build_infos"),
+    [
+        pytest.param(
+            ["amd64"],
```

Comment:
what about the case where `architectures` is `None`? I think I lost track of where we landed with regards to remote-build in this case.

Reply:
Added! In this case the remote build will retain its current behaviour (which was already tested for) of building for the host platform.